### PR TITLE
Externalize javascript: sms chat

### PR DESCRIPTION
**Pull request by orangejenny**
@gcapalbo / @jmtroth0 
cc @calellowitz 

Tested on staging that the chat window loads without error.

### CODE REVIEW

File: corehq/apps/sms/static/sms/js/chat.js [hunk 1]
```diff
@@ -0,0 +1,303 @@
+hqDefine("sms/js/chat", function() {
+    $(function() {
+        var initialPageData = hqImport("hqwebapp/js/initial_page_data");
+        $(function() {
```
Comment (jmtroth0) [inline corehq/apps/sms/static/sms/js/chat.js]: are both doc ready handlers necessary?

**Review by jmtroth0 — APPROVED** [review]

File: corehq/apps/sms/static/sms/js/chat.js (hunk 1 above)
Comment (orangejenny) [reply corehq/apps/sms/static/sms/js/chat.js]: oops, fixed

**Review by gcapalbo — APPROVED** [review]
@orangejenny found and fixed a couple minor things when testing locally

**Comment by orangejenny** [comment]
@gcapalbo thank you for fixing

**Comment by orangejenny** [comment]
Linted locally, there are a bunch of warnings on chat.js but they all look pre-existing and I'm not going to deal with them in this PR.

**Pull request merged**